src/Sylius/Bundle/CartBundle/spec/Form/Type/CartItemTypeSpec.php

```diff
@@ -37,12 +38,18 @@ function it_is_a_form_type()
         $this->shouldImplement(FormTypeInterface::class);
     }
 
-    function it_builds_form_with_quantity_field(FormBuilder $builder)
+    function it_builds_form_with_quantity_field($orderItemQuantityDataMapper, FormBuilder $builder)
     {
         $builder
             ->add('quantity', 'integer', Argument::any())
+            ->willReturn($builder)
             ->shouldBeCalled()
```

Comment:
I think we discussed this many times, we agree that these are redundant except for form/query builders. Otherwise specs for these are pointless. Agree?